src/cljs/ataru/virkailija/application/handlers.cljs

```diff
@@ -24,6 +24,12 @@
         (assoc-in db [:application :applications] (:applications aplications-response))))
     db))
 
+(reg-event-db
+ :application/review-updated
+ (fn [db [_ response]]
+   (assoc-in db [:application :review] (:review response))
```

Comment:
Is there a bug here? The `(assoc-in db [:application :review] (:review response))` will never be set to the app state. Only setting of the `(:events response)` below will be set there.